dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -56,6 +56,14 @@ const shouldInsertMerchHigh = (
 	return index === desiredPosition;
 };
 
+const isToggleable = (index: number, collection: DCRCollectionType) => {
+	return (
+		index != 0 &&
+		collection.collectionType != 'nav/list' &&
```

Comment:
I have never seen a `nav/list` or `nav/media/list`, especially in the first position of a front, but have moved this logic into DCR. Open to take this out.

Reply:
As per my comment, maybe consistency is better than having an arbitrary rule that readers need to some how reason about?